scripts/linters/general_purpose_linter.py

```diff
@@ -166,6 +166,7 @@
         'excluded_files': (
             'core/templates/pages/exploration-player-page/'
             'FeedbackPopupDirective.js',
+            '.component.ts'
```

Comment:
@srijanreddy98, Can you explain the reason for this change?

Reply:
Because angular components use templateUrl. If you look at the webpack configurations change, I added a loader to be able to use templateUrl in angular components.